streams/src/test/java/org/apache/kafka/streams/processor/internals/NamedTopologyTest.java

```diff
@@ -149,7 +149,7 @@ public void shouldThrowTopologyExceptionWhenMultipleNamedTopologiesCreateStreamF
     }
 
     @Test
-    public void shouldThrowTopologyExceptionWhenAddingNamedTopologyReadingFromSameInputTopic() {
+    public void shouldThrowTopologyExceptionWhenAddingNamedTopologyReadingFromSameInputTopicAfterStart() {
```

Comment:
Nit: After start?

Reply:
Can you elaborate? I did add a "AfterStart" suffix here -- is that what you meant?

Reply:
Sorry that was a bit cryptic. After start as a suffix doesn't really make sense to me as we never call start. Maybe "before start"?